### PR TITLE
Condrashov Serghei second java task

### PR DESCRIPTION
**Pull request by SerjCo84**

### CODE REVIEW

File: src/main/java/serghei_condrasov/ConditionMain.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            int number = scan.nextInt();
+            if (number % 2 == 0 && number % 3 == 0)
+                System.out.println("number is divisible by both 2 and 3");
+            else if (number % 2 == 0 || number % 3 == 0)
+                System.out.println("number is divisible by both 2 and 3");
+            else System.out.println("number is not divisible by both 2 and 3");
+
+        }
+        // 3. Determine the grade based on a given score.
+        public static void DetermineGrade() {
```
Comment (dgrimaliuc-crunchyroll) [inline src/main/java/serghei_condrasov/ConditionMain.java:36]: method should start from lowercase letter d
change DetermineGrade to determineGrade

File: src/main/java/serghei_condrasov/ConditionMain.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+import java.util.Scanner;
+public class ConditionMain {
+    public static void main(String[] args) {
+        DefinitionNumber ();
+        DivisibleNumber();
+        DetermineGrade();
+    }
+        //1. Check if a number is positive, negative, or zero.
+        public static void DefinitionNumber() {
```
Comment (dgrimaliuc-crunchyroll) [inline src/main/java/serghei_condrasov/ConditionMain.java:11]: same

File: src/main/java/serghei_condrasov/ConditionMain.java [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+            if (num > 0) {
+                System.out.println("number is positive");
+            } else if (num < 0) {
+                System.out.println("number is negative");
+            } else {
+                System.out.println("number is zero");
+            }
+    }
+        // 2. Check if a number is divisible by both 2 and 3.
+        public static void DivisibleNumber() {
```
Comment (dgrimaliuc-crunchyroll) [inline src/main/java/serghei_condrasov/ConditionMain.java:24]: same

File: src/main/java/serghei_condrasov/ConditionMain.java (hunk 2 above)
Comment (SerjCo84) [reply src/main/java/serghei_condrasov/ConditionMain.java]: fixed, please check

File: src/main/java/serghei_condrasov/ConditionMain.java (hunk 3 above)
Comment (SerjCo84) [reply src/main/java/serghei_condrasov/ConditionMain.java]: fixed, please check

**Review by dgrimaliuc — APPROVED** [review]

**Pull request merged**